src/winetricks

```diff
@@ -4675,6 +4675,16 @@ helper_win7sp1()
     w_try_cabextract -d "$W_TMP" -L -F "$filename" "$W_CACHE"/win7sp1/windows6.1-KB976932-X86.exe
 }
 
+helper_win7sp1_x64()
+{
+    filename=$1
+
+    # https://www.microsoft.com/en-us/download/details.aspx?id=5842
+    w_download_to win7sp1 https://download.microsoft.com/download/0/A/F/0AFB5316-3062-494A-AB78-7FB0D4461357/windows6.1-KB976932-X64.exe 74865ef2562006e51d7f9333b4a8d45b7a749dab
```

Comment:
sha256 would be welcome here:
`f4d1d418d91b1619688a482680ee032ffd2b65e420c6d2eaecf8aa3762aa64c8`